Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -770,6 +770,13 @@ def route_list_values() -> Response:
             request_args=request_args
         )
 
+        if demisto.getIntegrationContext() == {} and params.get('on_demand'):
```

Comment:
```suggestion
        if demisto.getIntegrationContext() and params.get('on_demand'):
```
would do the same isnt it?